core/src/main/scala/org/broadinstitute/dsp/workbench/welder/model.scala

```diff
@@ -68,10 +67,12 @@ final case class StorageLink(
     pattern: String
 )
 
+final case class HashedMetadata(asString: String) extends AnyVal
```

Comment:
shall we call this `hashedLockedBy` instead? cuz it's not hashing all metadata fields

Reply:
👍 I wanted it to be more generic because there were talks of having a lastEditedBy field which would also need to be hashed, but I don't think that's actually happening so this ended up being premature